static_inventories/copperfist_cluster.yml

```diff
@@ -258,4 +258,7 @@ copperfist_cluster:
     data_transfer:
     docs:
     smb_server:
+mit_chaperone:  # Special type of chaperone machine, which is partially managed by MIT.
+  children:
+    chaperone:
```

Comment:
Ok, the `mit_chaperone` segregates the managed vs unamanged chaperones - I get that.
But I don't get why you just assign all chaperones into `mit_chaperone`?
Wouldn't it be correct to have it

```suggestion
     cf-chaperone:
```
since not all chaperones are mit ones automatically. Only a subset of them are ...
(the same goes for wh and bb static inventories)

Reply:
Check, logic updated in recent commit.